examples/android/basic/settings.gradle.kts

```diff
@@ -8,12 +8,13 @@ pluginManagement {
 dependencyResolutionManagement {
   repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
   repositories {
-/* Used for internal testing of SDK releases
+  // Used for internal testing of SDK releases*/
+/*  
     maven {
-      url = uri("http://localhost:1480")
+      url = uri("http://adt-proxy.uplink2.goog:999/BB4d3gcrTY63P_93mtRMVw")
```

Comment:
Why is this being used? This localhost is to represent a local maven execution if you want to run it locally in a specific port. In this case port 1480. So it should be kept as is.